src/arch-sh.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * This library is free software; you can redistribute it and/or modify it
+ * under the terms of version 2.1 of the GNU Lesser General Public License as
+ * published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, see <http://www.gnu.org/licenses>.
+ */
+
+#include <stdlib.h>
+#include <errno.h>
+#include <linux/audit.h>
+
+#include "arch.h"
+#include "arch-sh.h"
+
+const struct arch_def arch_def_sheb = {
+	.token = SCMP_ARCH_SHEB,
+	.token_bpf = AUDIT_ARCH_SH,
+	.size = ARCH_SIZE_32,
+	.endian = ARCH_ENDIAN_BIG,
+	.syscall_resolve_name = sh_syscall_resolve_name,
+	.syscall_resolve_num = sh_syscall_resolve_num,
```

Comment:
I just want to double check this since I know very little about Super-H; the ***only*** difference between Super-H Big and Little Endian is the byte ordering?  The syscall numbering is exactly the same between the two?

Reply:
Yes, that's correct. SH has been bi-endian from the beginning, so the syscall numbers are the same.